test/convection/model/test_validation.rb

```diff
@@ -0,0 +1,184 @@
+require 'test_helper'
+require 'json'
+require 'pp'
+
+class TestValidations < Minitest::Test
+
+  def setup
+    @within_limits = ::Convection.template do
+      description 'Validations Test Template - Within Limits'
+      10.times do |count|
+        resource "EC2_INSTANCE_#{count}" do
+          type 'AWS::EC2::Instance'
+          property 'AvailabilityZone', 'us-east-1a'
+          property 'ImageId', 'ami-76e27e1e'
+          property 'KeyName', 'test'
+          property 'SecurityGroupIds', ['sg-dd733c41', 'sg-dd738df3']
+          property 'Tags', [{
+            'Key' => 'Name',
+            'Value' => 'test-1'
+            }]
+          end
+        end
```

Comment:
This `end`, the one above it, and the `}]`, are all intended two spaces too far.